app/Services/Generator.php

```diff
@@ -62,6 +63,19 @@ protected static function get_content_components($post) {
     return $components;
   }
 
+  protected static function get_footer_components(TimberPost $post) {
+    $components = [];
+    $theme_base_directory = get_stylesheet_directory();
+    $generator_class = $theme_base_directory . '/views/widgets/footer/generators/instant-articles/generator.php';
```

Comment:
@KimberleyCook Do you think this should be from the `views/partials` given what we discussed with @ecoad and @iamdeaneyelid about the header and footer not being widgets anymore? Can we even do that without the refactor? I do not have the answers, let's have a think.